src/PhpFlo/Core/Tests/Interaction/PortRegistryTest.php

```diff
@@ -8,13 +8,14 @@
  * file that was distributed with this source code.
  */
 
-namespace Tests\PhpFlo\Interaction;
+namespace Tests\PhpFlo\Core\Interaction;
 
 use PhpFlo\Common\PortInterface;
-use PhpFlo\Interaction\ArrayPort;
-use PhpFlo\Interaction\PortRegistry;
+use PhpFlo\Core\Interaction\ArrayPort;
+use PhpFlo\Core\Interaction\PortRegistry;
+use PhpFlo\Core\Test\TestCase;
 
-class PortRegistryTest extends \PHPUnit_Framework_TestCase
+class PortRegistryTest extends TestCase
```

Comment:
The thought behind that is to have a single class with the direct phpunit dependency for the v6.x refactoring later on, where phpunit switches to namespacing.